src/it/java/teammates/it/ui/webapi/DeleteStudentActionIT.java

```diff
@@ -0,0 +1,129 @@
+package teammates.it.ui.webapi;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.util.Const;
+import teammates.common.util.HibernateUtil;
+import teammates.storage.sqlentity.Course;
+import teammates.storage.sqlentity.Instructor;
+import teammates.storage.sqlentity.Student;
+import teammates.ui.webapi.DeleteStudentAction;
+
+/**
+ * SUT: {@link DeleteStudentAction}.
+ */
+public class DeleteStudentActionIT extends BaseActionIT<DeleteStudentAction> {
+
+    @Override
+    @BeforeMethod
+    protected void setUp() throws Exception {
+        super.setUp();
+        persistDataBundle(typicalBundle);
+        HibernateUtil.flushSession();
+    }
+
+    @Override
+    String getActionUri() {
+        return Const.ResourceURIs.STUDENT;
+    }
+
+    @Override
+    String getRequestMethod() {
+        return DELETE;
+    }
+
+    @Test
+    @Override
+    protected void testExecute() throws Exception {
+        Instructor instructor = typicalBundle.instructors.get("instructor1OfCourse1");
+        Student student1InCourse1 = typicalBundle.students.get("student1InCourse1");
+        Student student2InCourse1 = typicalBundle.students.get("student2InCourse1");
+
+        ______TS("Typical Success Case delete a student by email");
```

Comment:
Seems like none of the success cases actually check for anything. Checked the old datastore tests and this was the case as well, this means that tests for delete student action never actually tested anything this whole time :/

Shall we take this opportunity to fix the tests? Likely adding some `assertNull`s for success cases and the opposite for fails silently cases will do. Same case for `DeleteStudentsActionIT`.

Reply:
Thanks jingjing, yes, let's do that! Is it possible to access methods in the Logic layer to test? I wanted to do that the previous time but wasn't sure, as we're testing for the action but not any other things.

Reply:
Yeah should be able to use logic layer methods. Can check how it's done in `DeleteFeedbackQuestionActionIT` or `DeleteFeedbackResponseCommentActionIT`.